middleware/canCreateManualDonation.js

```diff
@@ -0,0 +1,17 @@
+const ErrorMessage = require("../constants/error");
+
+exports.canCreateManualDonation = async function (req, res, next) {
```

Comment:
This has already been merged before. Why is it showing up here again? @Chiemezuo